renta_facil_colima/src/hooks/GetMyHousesFromFirebase.js

```diff
@@ -0,0 +1,58 @@
+import {initializeApp} from 'firebase/app';
+import {firebaseConfig} from '../Firebase/FirebaseConfig';
+import {Alert} from 'react-native';
+import {
+  getFirestore,
+  collection,
+  doc,
+  getDocs,
+  getDoc,
+  query
+} from 'firebase/firestore/lite';
+
+const GetMyHousesFromFirebase = () => {
+  initializeApp(firebaseConfig);
+  const db = getFirestore();
+
+  const GetMyHouses = async (uid) => {
+    try {
+        const q = query(collection(db, 'Houses'), where("uid", "==", uid));
+    
+      const dataCollection = await getDocs(q);
+      const data = [];
+      dataCollection.forEach(documentSnapshot => {
+        const item = {
+          id: documentSnapshot.id,
+          ...documentSnapshot.data(),
+        };
+        data.push(item);
+      });
+      return data;
+    } catch (error) {
+      Alert.alert('Error', `We could not get the ${collectionName} data`);
+    }
+  };
```

Comment:
We talked about to migrate all hooks  firebase firestore hooks from firebase package to react-native-firebase/firestore and create new with react-native-firebase/firestore, please change the package [as we have in our other hooks on main](https://github.com/BrightCoders-Institute/proyecto-final-team1a/blob/main/renta_facil_colima/src/hooks/GetDataFromFirebase.js)